openslides_backend/action/actions/list_of_speakers/re_add_last.py

```diff
@@ -68,11 +68,11 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
         for speaker in speakers.values():
             if (
                 speaker.get("end_time") is None
-                and speaker["user_id"] == last_speaker["user_id"]
+                and speaker["meeting_user_id"] == last_speaker["meeting_user_id"]
                 and not speaker.get("point_of_order")
             ):
                 raise ActionException(
-                    f"User {last_speaker['user_id']} is already on the list of speakers."
+                    f"Meeting User {last_speaker['meeting_user_id']} is already on the list of speakers."
```

Comment:
Please read the user_id and give the old error message! The meeting_user_id don't really help the user/speaker-list admin

Reply:
Okay, read the meeting_user and return the user_id here.